client/src/manifest.json

```diff
@@ -34,6 +34,8 @@
     "method": "POST",
     "enctype": "multipart/form-data",
     "params": {
+      "title": "Blaze",
```

Comment:
@cristicismas the key value for title and text should not be "Blaze". The key value pair under params determines the query parameters and most of the applications use "title" for "title" and "text" for "text".

Reply:
Thanks for the review @abhishek-aa :smile: